modules/serverless-pingsource-ODC.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-pingsource.adoc
+
+[id="serverless-pingsource-ODC_{context}"]
+= Creating a PingSource using the Developer Perspective
+
+The following sections describe how to create and verify a basic PingSource from the {product-title} web console.
+
+.Prerequisites
+To create a PingSource using the *Developer* perspective ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console.
+* You are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. To verify that the PingSource is working, create a simple Knative
+service that dumps incoming messages to the service's logs.
```

Comment:
s/service's logs/logs of the service

To follow guidelines for avoiding inanimate object possession.